mava/wrappers/jumanji.py

```diff
@@ -222,15 +227,20 @@ class ConnectorWrapper(JumanjiMarlWrapper):
     Do not use the AgentID wrapper with this env, it has implicit agent IDs.
     """
 
-    def __init__(self, env: MaConnector, add_global_state: bool = False):
+    def __init__(self, env: Connector, add_global_state: bool = False):
         super().__init__(env, add_global_state)
-        self._env: MaConnector
+        self._env: Connector
 
     def modify_timestep(self, timestep: TimeStep) -> TimeStep[Observation]:
         """Modify the timestep for the Connector environment."""
 
         # TARGET = 3 = The number of different types of items on the grid.
         def create_agents_view(grid: chex.Array) -> chex.Array:
+            # add switch_perspective to change grid (@sasha-a what do you think?):
```

Comment:
Please remove this comment. Also @sash-a can you please comment here?

Reply:
@sash-a I explained it all below 😅